packages/cf-component-pagination/src/Pagination.js

```diff
@@ -1,31 +1,53 @@
 import React, { PropTypes } from 'react';
 import uniqueId from 'lodash/uniqueId';
+import { createComponent } from 'cf-style-container';
+import PaginationRoot from './PaginationRoot';
+import PaginationTheme from './PaginationTheme';
+import { applyTheme } from 'cf-style-container';
 
-class Pagination extends React.Component {
-  render() {
-    const id = this.props.info ? uniqueId('cf-pagination-') : null;
+const styles = ({ theme }) => ({
+  '&:after': {
+    content: theme.content,
+    display: theme.display,
+    clear: theme.clear
+  },
+  listStyle: theme.listStyle,
+  marginTop: theme.marginTop,
+  marginLeft: theme.marginLeft,
+  marginRight: theme.marginRight,
+  marginBottom: theme.marginBottom,
+  paddingTop: theme.paddingTop,
+  paddingLeft: theme.paddingLeft,
+  paddingRight: theme.paddingRight,
+  paddingBottom: theme.paddingBottom,
+  float: theme.float,
+  borderRadius: theme.borderRadius,
+  boxShadow: theme.boxShadow
+});
 
-    return (
-      <div className="cf-pagination">
-        <ul
-          className="cf-pagination__list"
-          role="navigation"
-          aria-describedby={id}
-        >
-          {this.props.children}
-        </ul>
-        {this.props.info &&
-          <span id={id}>
-            {this.props.info}
-          </span>}
-      </div>
-    );
-  }
-}
+const Pagination = ({ info, children, className }) => {
+  const id = info ? uniqueId('cf-pagination-') : null;
+  return (
+    <PaginationRoot>
+      <ul className={className} role="navigation" aria-describedby={id}>
+        {children}
+      </ul>
+      {info &&
+        <span id={id}>
+          {info}
+        </span>}
+    </PaginationRoot>
+  );
+};
 
 Pagination.propTypes = {
+  className: PropTypes.string.isRequired,
   info: PropTypes.string,
   children: PropTypes.node
 };
 
-export default Pagination;
+const PaginationUnstyled = createComponent(styles, Pagination);
+
+export { PaginationUnstyled };
+
+export default applyTheme(PaginationUnstyled, PaginationTheme);
```

Comment:
I want consistency with my code, I use default export for PaginationLink and PaginationRoot